actions/crib-deploy-environment/action.yml

```diff
@@ -128,8 +128,21 @@ outputs:
 runs:
   using: "composite"
   steps:
+    - name: Generate CRIB Namespace Name and labels
+      id: generate-ns-name
+      shell: bash
+      run: |
+        echo "devspace-namespace=${{ inputs.ns-name-prefix }}-$(uuidgen | cut -c1-5)" | tee -a "$GITHUB_OUTPUT"
+        # This will work only for PRs, push event itself does not contain PR information.
+        echo "pr-number=${{ github.event.number }}" | tee -a "$GITHUB_OUTPUT"
+        echo "workflow-job=${{ github.job }}" | tee -a "$GITHUB_OUTPUT"
+        echo "repo-name=$(basename $GITHUB_REPOSITORY)" | tee -a "$GITHUB_OUTPUT"
+        echo "branch-name=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" | tee -a "$GITHUB_OUTPUT"
+        echo "commit-sha=${{ github.sha }}" | tee -a "$GITHUB_OUTPUT"
+        echo "workflow-run-number=${{ github.run_number }}" | tee -a "$GITHUB_OUTPUT"
```

Comment:
Should see about using env's to avoid template injection too: https://woodruffw.github.io/zizmor/audits/#template-injection